requirements.txt

```diff
@@ -16,3 +16,4 @@ tzdata==2024.1
 tzlocal==5.2
 Werkzeug==3.0.3
 WTForms==3.1.2
+autopep8==2.3.1
```

Comment:
This isn't required by the application, is it? Perhaps it makes more sense to add the installation in the `README.md`, but don't require it for the application itself

Reply:
Removed from requirements and added installation steps to the README.md